src/Management/test/Endpoint.Test/Actuators/CloudFoundry/EndpointMiddlewareTest.cs

```diff
@@ -85,6 +93,10 @@ public async Task CloudFoundryEndpointMiddleware_ServiceContractNotBroken()
                   "href": "http://localhost/cloudfoundryapplication/info",
                   "templated": false
                 },
+                "httptrace": {
+                  "href": "http://localhost/cloudfoundryapplication/httptrace",
+                  "templated": false
+                },
```

Comment:
Was this added intentionally? It looks like it doesn't belong here, but perhaps I misunderstood.

Reply:
This was added as a cf-specific integration test, it's irrelevant now